docs/skillmap/space/activity3.md

```diff
@@ -2,9 +2,70 @@
 
 ### @autoexpandOff true
```

Comment:
autoexpandOff was removed in activity1 and activity2, is it intentionally left here?

Reply:
As students progress, I'm thinking they'll go to instructions as needed.  But it looks like they don't even know they're there when they first start, so if we leave instructions expanded for the first 1/2 it's more obvious.

@shakao